codespell_lib/_codespell.py

```diff
@@ -245,7 +244,7 @@ def open_with_chardet(self, filename: str) -> Tuple[List[str], str]:
             lines = f.readlines()
             f.close()
 
-        return lines, encoding
+        return lines, f.encoding
```

Comment:
Excellent, we return [io.TextIOBase](https://docs.python.org/3/library/io.html#io.TextIOBase).[encoding](https://docs.python.org/3/library/io.html#io.TextIOBase.encoding), as actually set by [open](https://docs.python.org/3/library/functions.html#open) in text mode (the default).